### ISSUE TITLE
Incorrect Translation in Simplified Chinese UI

### ISSUE THREAD

**Issue by junkworld** [brave/brave-browser#37905]
<code style="background-color: #f0f0f0;">New private window with tor</code> should be translated as <code style="background-color: #f0f0f0;">用Tor打开新的无痕窗口</code>,
not <code style="background-color: #f0f0f0;">用Tor打开新的私人窗口</code>.

Also
Settings - Privacy and Security - tor window - <code style="background-color: #f0f0f0;">private window with tor</code> should be translated as <code style="background-color: #f0f0f0;">带tor的无痕窗口</code>,
not <code style="background-color: #f0f0f0;">带tor的私人窗口</code>.

Settings - Sheilds - Trackers & ads blocking - option <code style="background-color: #f0f0f0;">Aggressive</code> should be translated as <code style="background-color: #f0f0f0;">积极地</code> or <code style="background-color: #f0f0f0;">主动</code>, not <code style="background-color: #f0f0f0;">攻击性</code>.

Settings - Sheilds - Trackers & ads blocking - <code style="background-color: #f0f0f0;">Allow Facebook logins and embedded posts</code> should be translated as <code style="background-color: #f0f0f0;">允许Facebook登陆和嵌入帖子</code>.

Settings - Sheilds - Trackers & ads blocking - <code style="background-color: #f0f0f0;">Forget me when I close this site</code> should be translated as <code style="background-color: #f0f0f0;">关闭此网站后遗忘我</code>, not <code style="background-color: #f0f0f0;">关闭此网站后勿扰</code>.
In Chinese, <code style="background-color: #f0f0f0;">勿扰</code> means <code style="background-color: #f0f0f0;">do not disturb</code> .

Settings - Appearance - Tabs - Tab hover mode, option <code style="background-color: #f0f0f0;">card</code> should be translated as <code style="background-color: #f0f0f0;">卡片</code>, not <code style="background-color: #f0f0f0;">信用卡</code>.
In Chinese, <code style="background-color: #f0f0f0;">信用卡</code> means <code style="background-color: #f0f0f0;">credit card</code> .

**Comment by rebron** [comment]
Thanks @junkworld 

@mkarolin. @larreewong reviewed and the suggestions here are good.

**Comment by mkarolin** [comment]
@rebron

> "New private window with tor" should be translated as "用Tor打开新的无痕窗口",
not "用Tor打开新的私人窗口" .

Google translate gives me `Open new private window with Tor` for the current translation, but for the suggested translation I get `Open a new incognito window with Tor`.

> Also
Settings - Privacy and Security - tor window - "private window with tor " should be translated as "带tor的无痕窗口",
not "带tor的私人窗口".

Similarly, for our current translation Google translate gives me `Private window with tor`, while the suggested translation shows as `Incognito windows with tor`.

**Comment by rebron** [comment]
Let's stick with our `private` language vs `incognito`.

**Comment by mkarolin** [comment]
Updated the rest in Transifex. Should get them for the upcoming 1.66.x release.

**Comment by junkworld** [comment]
> Let's stick with our `private` language vs `incognito`.

Translate "private window" as "隐私窗口“ is a more correct translation,
In Chinese,"私人" means more "Personal","私人"does not emphasize privacy.

The current situation is that there are too many "private window" words in the brave settings. Some are translated as “无痕窗口" and some are translated as "私人窗口". 

They are all the same words "private window" and have not been unified translated into the same Chinese words.
![图像](https://github.com/brave/brave-browser/assets/108603549/c4897584-3d39-40f2-8598-1ae8bbed5122)
![图像-2](https://github.com/brave/brave-browser/assets/108603549/d104ab15-2519-4c41-872f-be28b346441f)
![图像-3](https://github.com/brave/brave-browser/assets/108603549/4573b150-0679-49da-83a7-419f8d56f276)
![图像-4](https://github.com/brave/brave-browser/assets/108603549/8fe358d7-98bb-4f4a-81f2-d965dc2c05fb)


"Incognito window"/"无痕窗口“ is used in Chromium, I guess since Brave is a fork of Chromium, Brave inherited a lot of the translations in Chromium
![图像-5](https://github.com/brave/brave-browser/assets/108603549/5659e843-d3b5-4bfd-a164-0187de977df6)

Firefox translate “private window" as "隐私窗口"
![图像-6](https://github.com/brave/brave-browser/assets/108603549/3ea5bf85-7212-40b1-a0b5-8fc3f609792e)

**Comment by junkworld** [comment]
settings - system -shorcuts

There is a letter J missing here

![图像-7](https://github.com/brave/brave-browser/assets/108603549/7a923615-e749-42e3-b28f-754fea51be97)

**Comment by junkworld** [comment]
in extensions page
"Web Store" should be translated just as "应用商店" ,not "Brave 网上应用店" 



![图像-8](https://github.com/brave/brave-browser/assets/108603549/4266fb8c-85e5-4786-a072-8634f97a31fd)
<img width="503" alt="截屏2024-05-01 19 00 21" src="https://github.com/brave/brave-browser/assets/108603549/85fce256-3cd4-4689-aa13-520b3f0daddf">

**Comment by mkarolin** [comment]
Thank you @junkworld.

I've updated the `Web Store` strings.

With Javascript, that looks like we maybe stripping the leading `J` programmatically by mistake (cc: @fallaciousreasoning - the translated string is `JavaScript 控制台(&amp;J)` in `app/resources/generated_resources_zh-CN.xtb`. We are probably removing the leading `J` by accident when we strip `&amp;J` from the translation.)

The private vs incognito issue is not limited to just the Chinese translations. It sounds like we really need a more comprehensive effort to clean up the interchangeable use of these terms in translations @rebron, @AndyAnds265.

**Comment by junkworld** [comment]
Hi,
I think in Appearance the first "Brave colors" should be named as "Brave mode",
Which matches the system's light mode and dark mode.
There are two "Brave colors",kind of confused.


"Brave mode" should be translated as"Brave 模式" in Chinese, 
Of course "Brave mode" should also be translated into any other languages.

<img width="713" alt="截屏2024-05-02 20 29 44" src="https://github.com/brave/brave-browser/assets/108603549/b25188e7-d106-476c-8114-101dc5026485">
<img width="713" alt="截屏2024-05-02 20 30 12" src="https://github.com/brave/brave-browser/assets/108603549/4621e8f2-912c-429a-a51b-535667c8e94c">
<img width="713" alt="截屏2024-05-02 20 39 57" src="https://github.com/brave/brave-browser/assets/108603549/42eef907-06d6-43e5-9e52-1147e0d52ae2">

**Comment by junkworld** [comment]
![图像](https://github.com/brave/brave-browser/assets/108603549/13a9224b-8cf5-4149-8ed8-c9e2205c630c)


In Customize Fonts page,there is a mispalced white dot and a misplaced red dot for unknown purpose

**Comment by junkworld** [comment]
Hi, @mkarolin 

in right click panel
"copy clean link" should be translated as "复制干净的链接" or "复制无跟踪参数链接",
not "拷贝清除链接" , "清除" means "clear away".
"复制无跟踪参数链接" is the a more appropriate translation that accurately describes this feature in Chinese context

![图像-2](https://github.com/brave/brave-browser/assets/108603549/09550500-cb44-444e-bc5d-06d242c49603)

**Comment by junkworld** [comment]
vertical tabs - "new tab below" should be translated as "在下方新增标签页"

it will be unified in format with "在右侧新增标签页"/"new tab to the right"

"下列新标签页" in Chinese context is more like "move this tab down"

![图像](https://github.com/brave/brave-browser/assets/108603549/89147088-ce21-40a8-9561-02d8089e9a81)


![图像-2](https://github.com/brave/brave-browser/assets/108603549/caf7aea4-f301-44de-b16a-e61c1bdcab3a)

**Comment by junkworld** [comment]
Hi,
I'm already using the 1.66.110 stable version, 
and I'm glad to see that many of the issues have been fixed. 
Except for those that are still not finalized, 
there are still some issues that have not been fixed.

Get started - "New tab page shows" should be translated as "新标签页显示",not "新标签页随及会显示"
"随及会" three words are grammatically incorrect in Chinese context,confusing and redundant，should be deleted.

Control panel - "Brave 的 VPN" , "的" should be deleted, when "的" is added,that would change its meaning to "Brave's VPN","Brave's VPN" and "Brave VPN" have completely different meanings.

Privacy and security - Tor windows - "Private window with Tor"  windows should be translated as "窗口", not "窗户".
All these "window" and "windows" should be translated as "窗口",somehow,this one is strangely translated as "窗户".
In Chinese context, we use "窗户" to refer to a physical window in a house. This word cannot be used to refer to a window in a computer operating system.
![图像](https://github.com/brave/brave-browser/assets/108603549/4cee3bdd-04a5-4923-a236-af778e45bbce)

**Comment by junkworld** [comment]
system - shortcuts , duplicate tab, pin tab, group tab, double quotation mark (“ ”)were added by mistake, these double quotation mark should be deleted.
"添加" and "重置" Add and reset should be arranged horizontally,not arrange vertically

<img width="658" alt="截屏2024-05-18 15 06 05" src="https://github.com/brave/brave-browser/assets/108603549/c3a67f3e-79ae-4519-bce3-f031ae5bc037">

**Comment by kjozwiak** [comment]
Closing the above as https://github.com/brave/brave-core/pull/23735 was merged into `1.66.x`. The above requires `1.66.112` or higher for `1.66.x` verification 👍

**Comment by GeetaSarvadnya** [comment]
Verification INPROGRESS on

![image](https://github.com/brave/brave-browser/assets/38657976/99e7121e-ed9b-4ef9-976a-1045bac41a56)

**Comment by junkworld** [comment]
Verification PASSED on
<img width="836" alt="截屏2024-05-23 14 09 15" src="https://github.com/brave/brave-browser/assets/108603549/6fd8845f-2bd0-44e4-ac39-0f13aadd3f6e">

---

Verified the Simplified Chinese translations from the description 
https://github.com/brave/brave-browser/issues/37905#issuecomment-2117407507


Get started - <code style="background-color: #f0f0f0;">New tab page shows</code> should be translated as <code style="background-color: #f0f0f0;">新标签页显示</code>,not <code style="background-color: #f0f0f0;">新标签页随及会显示</code>
 **PASSED**

<img width="702" alt="截屏2024-05-23 14 10 13" src="https://github.com/brave/brave-browser/assets/108603549/3c7980de-eaac-41cd-b24f-d02b2b5a7d37">

Control panel - <code style="background-color: #f0f0f0;">Brave 的 VPN</code> , <code style="background-color: #f0f0f0;">的</code> should be deleted
**PASSED**
Settings - System - Shortcuts ,These <code style="background-color: #f0f0f0;">的</code> should also be deleted
**PASSED**
<img width="702" alt="截屏2024-05-23 14 11 33" src="https://github.com/brave/brave-browser/assets/108603549/6b557473-defb-4350-afca-626cdb38cac5">
<img width="702" alt="截屏2024-05-23 14 11 45" src="https://github.com/brave/brave-browser/assets/108603549/2e167d4f-5f11-41eb-98aa-b6aab51a28d7">

Privacy and security - Tor windows - <code style="background-color: #f0f0f0;">Private window with Tor</code>  window should be translated as <code style="background-color: #f0f0f0;">窗口</code>, not <code style="background-color: #f0f0f0;">窗户</code>
 **PASSED**
<img width="702" alt="截屏2024-05-23 14 10 27" src="https://github.com/brave/brave-browser/assets/108603549/e873f13c-f824-4d42-bb97-182650ab8aac">

---


Verified the Simplified Chinese translations from the description
https://github.com/brave/brave-browser/issues/37905#issuecomment-2119142084

System - Shortcuts , 
<code style="background-color: #f0f0f0;">Duplicate tab/复制标签页</code>
<code style="background-color: #f0f0f0;">Pin tab/固定标签页</code>
<code style="background-color: #f0f0f0;">Group tab/组标签页</code>
double quotation mark (“ ”)were added by mistake, these double quotation mark(“ ”) should be deleted. 
**PASSED**
<img width="702" alt="截屏2024-05-23 14 11 55" src="https://github.com/brave/brave-browser/assets/108603549/a3a10a43-3e9b-42ba-bb85-f536c0852bca">
<img width="678" alt="截屏2024-05-23 14 12 24" src="https://github.com/brave/brave-browser/assets/108603549/8b336855-4f39-4a46-aa05-f20f615b6de4">
<img width="678" alt="截屏2024-05-23 14 12 36" src="https://github.com/brave/brave-browser/assets/108603549/df4587d6-9337-4812-bf7c-83e776f1dda4">

<code style="background-color: #f0f0f0;">Group tab</code> should be translated as  <code style="background-color: #f0f0f0;">分组标签页</code>, not  <code style="background-color: #f0f0f0;">组标签页</code>
 **NOT PASSED**
<img width="702" alt="截屏2024-05-23 14 11 55" src="https://github.com/brave/brave-browser/assets/108603549/ba30f5b4-7328-459f-adaf-e59d6f787496">

<code style="background-color: #f0f0f0;">添加</code>  and <code style="background-color: #f0f0f0;">重置</code> should be arranged horizontally,not arranged vertically 
**NOT PASSED**
<img width="178" alt="截屏2024-05-23 14 12 11" src="https://github.com/brave/brave-browser/assets/108603549/ed4a24f6-09b5-410d-b172-0195c990f556">

**Comment by mkarolin** [comment]
> `Group tab` should be translated as 分组标签页, not 组标签页
@junkworld, forgot to ask you about this .. the original string is for `tab` (not `tabs`). Google translate seems to indicate that your suggestion would mean `tabs` (plural).

> 添加 and 重置 should be arranged horizontally,not arranged vertically
This is more of a layout issue, rather than a translation issue, so we would need to address that separately. cc: @fallaciousreasoning

**Comment by junkworld** [comment]
> > `Group tab` should be translated as 分组标签页, not 组标签页
> > @junkworld, forgot to ask you about this .. the original string is for `tab` (not `tabs`). Google translate seems to indicate that your suggestion would mean `tabs` (plural).
> 
> > 添加 and 重置 should be arranged horizontally,not arranged vertically
> > This is more of a layout issue, rather than a translation issue, so we would need to address that separately. cc: @fallaciousreasoning

@mkarolin 

I'm a heavy user of tab groups,I set `Ctrl`+`G` for `Group tab` shortcut,
I found it has the same function as `Add tab to New Group`,no matter I select one tab, two tabs or more.
Are they the same function?

---

And let me explain why `组标签页` is incorrect. 
When the single word `组` is used as a verb in the Chinese context, there is usually only one usage of `组我`、`组你`or`组他`, which means to 'add me into the team','add you into the team' ,'add him into the team'. 
We don't add nouns after a single word `组`,this is a grammatically wrong sentence in Chinese context,and the plural is not applicable in this sentence in Chinese context,Google translate gives an incorrect translation.
The word `组` is usually used as part of compound words such as 
`分组`:grouping,divide into groups,sectionalization
`组织`:organize,organization,
`组装`:assemble,
`组合`:combination,
`组队`:form a team,
`组建`:establish,build

If `Group tab` has same function as `Add tab to New Group` ,it is better to use that same translation. or just use `分组标签页`
<img width="401" alt="截屏2024-05-23 22 16 11" src="https://github.com/brave/brave-browser/assets/108603549/735f0ec5-50d1-4b10-b429-5973f31a4bcf">
